foundation/application.go

```diff
@@ -206,3 +225,34 @@ func setRootPath() {
 
 	support.RootPath = rootPath
 }
+
+func getEnvPath() string {
```

Comment:
We don't use `flag` to get `env`, because custom "env" has problem while testing.